api/envoy/config/filter/network/kafka_broker/v2alpha1/kafka_broker.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package envoy.config.filter.network.kafka_broker.v2alpha1;
```

Comment:
nit: `v2alpha` (see https://github.com/envoyproxy/envoy/issues/8131)

Reply:
to be done later as per https://github.com/envoyproxy/envoy/issues/8131#issuecomment-559255960 (https://github.com/envoyproxy/envoy/blob/master/docs/root/api-v2/config/filter/network/network.rst needs to get fixed first)